runtime/verbose/CMakeLists.txt

```diff
@@ -80,6 +80,11 @@ target_link_libraries(j9vrb
 		j9gcvrbhdlrvlhgc
 		j9zip
 		j9utilcore
+		j9gc
+		j9gctrc
+		j9gcapi
+		j9gctrcstandard
+		j9gctrcvlhgc
```

Comment:
What are the implications of this for non AIX+OPENXL?

Reply:
Does this affect configs that are not AIX+OPENXL, shouldn't this also be moved to https://github.com/eclipse-openj9/openj9/pull/20799?

Reply:
Sorry, misunderstood the first time. The changes have been moved over!